Source/WTF/wtf/PlatformHave.h

```diff
@@ -1355,3 +1355,7 @@
     || ((PLATFORM(IOS) || PLATFORM(MACCATALYST)) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 160000)
 #define HAVE_CONTINUITY_CAMEARA 1
 #endif
+
+#if PLATFORM(MAC) && !PLATFORM(IOS_SIMULATOR) && __has_include(<apfs/apfs_fsctl.h>)
```

Comment:
Do we really need `!PLATFORM(IOS_SIMULATOR)` if we already have `PLATFORM(MAC)`?

Reply:
Also, any reason we're not doing this on iOS as well?

Reply:
ah nice catch, the first condition should be `PLATFORM(COCOA)`